<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTTCData</class>
 <widget class="QDialog" name="DialogTTCData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>810</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>810</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>META_START</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OBJECT_NAME">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>YH-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OBJECT_NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OBJECT_ID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2009-250B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OBJECT_ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OBJECT_CATNUM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OBJECT_CATNUM">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TIME_SYSTEM">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UTC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TIME_SYSTEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OBS_END_TIME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OBS_END_TIME">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2006-07-10T23:59:59.000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OBS_START_TIME">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2006-07-10T00:00:00.000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OBS_START_TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SC_TRANS_FRE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TTC_FAC_ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SC_TRANS_FRE">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8410010000.000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TTC_FAC_ID">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12301</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_station">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>861</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA_START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>820</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA_STOP</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_fileList">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>561</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="REC_REF_FRE">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8410000000.000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>REC_REF_FRE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TTC_TRANS_FRE">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7620000000.000 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TTC_TRANS_FRE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TURN_RATIO">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>880 : 749</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TURN_RATIO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TRS_START_TIME">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2006-07-10T12:00:00.000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TRS_START_TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TRS_END_TIME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TRS_END_TIME">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>220</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2006-07-10T23:59:59.000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addStation">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>290</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新增测量站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveStation">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>320</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delStation">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>350</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refreshStation">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>380</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新数据/取消修改</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OBJECT_CATNUM">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OBJECT_ID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OBJECT_NAME">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OBS_START_TIME">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_SC_TRANS_FRE">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OBS_END_TIME">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>820</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_REC_REF_FRE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_TTC_TRANS_FRE">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_TURN_RATIO">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_TRS_START_TIME">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_TRS_END_TIME">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>220</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_17">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>META_STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addFile">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新增测量文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delFile">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>600</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>航天器名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>航天器国际编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>航天器编目号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>观测时标的时间系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>观测开始时刻</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>观测结束时刻</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>航天器发射频率，
单位Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>测站编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>接收参考频率，单位Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>测站发射频率，单位Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>信号转发比</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>使用此发射频率的开始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>使用此发射频率的结束时间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
